README.md

```diff
@@ -1,90 +1,88 @@
+- [**Rules for Simple Placement of Japanese Ruby (ルビの簡便な配置ルール)**](https://www.w3.org/TR/simple-ruby/) • [*Editor's draft (編集草案)*](https://w3c.github.io/simple-ruby/) • [*Latest commits (直近の更新)*](https://github.com/w3c/simple-ruby/commits/gh-pages/index.html)
 
-The focus is especially on typographic features for which information in English is hard to find, such as justification, letter-spacing, vertical text, text decoration, page layout, emphasis, etc.   We want to ensure that we have captured local user needs in CSS, HTML, Timed Text, Web Payments, Web Publishing, and the many other specifications that the W3C produces. 
-
-議論において特に重要なのは英語で情報を得ることの難しい組版の機能についてです。例えば行揃え、文字間隔、縦書き、文字の修飾、ページ構成、強調の方法、等です。W3C は W3C が開発するさまざまな仕様書、例えば CSS, HTML, Timed Text, Web Payments, Web Publishing 等が、それぞれの地域で必要な機能を必ず取り込んでいるようにしたいと考えています。
-
-### Documents（文書）
-- [Japanese Gap Analysis](https://w3c.github.io/jlreq/gap-analysis/) （日本語ギャップ分析） 
-- [Requirements for Japanese Text Layout](https://w3c.github.io/jlreq/) (Editor's copy)
-- [Requirements for Japanese Text Layout](https://www.w3.org/TR/jlreq/) (Working Group Note)
-- [Requirements for Japanese Text Layout](https://www.w3.org/TR/jlreq/?lang=ja) (Japanese version、日本語版)
-- [Rules for Simple Placement of Japanese Ruby](https://w3c.github.io/simple-ruby/)
-
-### Related documents
+### Related documents (関連文書)
 - [Current Status of Japanese Typography Using Web Technologies](https://www.w3.org/Submission/2017/SUBM-CSJTUWT-20170102/)
 - [Ready-made Counter Styles](https://www.w3.org/TR/predefined-counter-styles/)
 
+
 ### Feedback
-Please use the [GitHub issue](https://github.com/w3c/jlreq/issues) to report issues for language support, for discussions, and to send feedback about documents. (Learn [how GitHub issues work](http://w3c.github.io/i18n-activity/guidelines/issues.html).)
+Please use the [GitHub issue list](https://github.com/w3c/jlreq/issues) to report issues for language support, for discussions, and to send feedback about documents. (Learn [how GitHub issues work](https://www.w3.org/International/i18n-activity/guidelines/issues.html).)
 
-言語の機能についての問題報告、議論、文書への意見については、[GitHub issue](https://github.com/w3c/jlreq/issues) にお願いします。([GitHub issues の利用](http://w3c.github.io/i18n-activity/guidelines/issues.html)についての文書を参考ください。)
+言語の機能についての問題報告、議論、文書への意見については、[GitHub issue](https://github.com/w3c/jlreq/issues) にお願いします。([GitHub issues の利用](https://www.w3.org/International/i18n-activity/guidelines/issues.html)についての文書を参考ください。)
 
-Note: The public-i18n-cjk mailing list is used to send notification digests & meeting minutes. It is **not** for technical discussion.
+Note that the public-jlreq-admin mailing list is used to send notification digests & meeting minutes. It is **not** for technical discussion.
```

Comment:
Notifications are not sent to the admin list, because it's not for technical discussion.

Reply:
ie. "public-jlreq-admin" should say "public-i18n-japanese".

Reply:
I assume that you are trying to get around the fact that members of the jlreq group are resisting the use of github issues and continue to discuss things by email.  The answer to that is to carefully manage the wording about use of that list – not to switch users to the admin list.

I suggest that, since we are unable to convince group members to follow the normal process, we replace "It is not for technical discussion." with "Some technical discussion takes place on that list for the Japanese language enablement, but it is strongly preferred to use GitHub issues whenever possible."